### DISCUSSION TITLE
조직에 첫 데이터 분석가가 필요한 시점은 언제일까요?

### DISCUSSION THREAD

**Discussion by jsaang** [playinpap/awesome-data-and-analytics-governance#12] [Q&A]
조직에 첫 데이터 분석가가 필요한 시점은 언제일까요?

**Comment by BokyungChoi** [comment]
안녕하세요, 
저도 조직 내 첫 데이터 분석가로 근무했던 경험이 있어 되새겨보며 답변을 남겨봅니다. 

## 이야기

제가 채용되었던 시점은, 
사내 데이터 팀이 `팀 리드 및 엔지니어링이 가능한 인력`을 갖추고, `분석을 할 수 있는 어느 정도의 환경`이 세팅된 뒤였습니다. 

이 때, `분석을 할 수 있는 어느 정도의 환경` 은 정의하기 나름이겠지만요.
제가 오기 전에 준비되었던 환경은,

- Amazon Redshift 를 통해서 데이터를 추출할 수 있음
- 로깅은 이미 전부터 진행해서 로그들이 남아 있음, 하지만 제대로 활용되지 않아서 유지 / 보수가 안 되는 것도 많고 컨벤션도 모두 다름
- 클라이언트 로그의 경우 데이터 마트가 없고 날 것의 데이터만 존재함
- 사내 데이터 엔지니어, 기획자, 개발자 분들이 세팅해 둔 BI Dashboard가 있는 상황
- 사내에서 실험, 지표에 관심이 많이 있는 편인 상황 (이 점이 참 행운이었네요)

다른 상황에 계셨던 지인 분들을 만나보면 저는 비교적으로 아름다운 환경이었던 것 같습니다.
따라서 위 조건들이 갖춰져 있지 않더라도 분석가를 모시는 것은 충분히 선행될 수 있다고 생각하는데요.
최소한의 조건을 아래와 같이 정리해보았습니다.

## 분석가를 위해 최소한 갖춰져야 하는 환경 조건
- 스몰 데이터가 아닌, 파이프라인 상으로 들어오는 데이터가 존재함 (서버, 클라이언트)
- 서버, 클라이언트 데이터를 SQL을 통해 직접 추출할 수 있는 환경이 구비되어 있음 
- 경영진 레벨에서 지표, 데이터 분석에 대한 중요성을 확실히 인지함

최소한으로 갖춰져야 하는 조건은 위와 같고, 무엇보다 드리고 싶은 이야기는요.
조직 - 분석가 개인이 모두 win-win 하기 위해서는 채용하시게 될 개인이 어떤 강점과 관심 영역을 지니고 계신 분인지 파악하셔야 합니다.

데이터 분석가 중에서도, 예를 들어 아래와 같은 페르소나가 있다고 해볼게요. 
1. 데이터 모델링에 능숙하며 관심이 있음 
2. 데이터 분석을 통해 인사이트를 도출하는 것에 능숙하며 관심이 있음 
3. 데이터 엔지니어링에 관심이 있고 역량 개발해보고 싶음 
4. 지표 정의에 진심이고, 상당히 꼼꼼히 집계하며 다양한 도메인의 지표에 능숙함
5. 새로운 분석 기술에 관심이 있고 역량 개발해보고 싶음

이 중에서, 위 최소 조건이 갖춰진 환경이라고 하면 1,3,4의 페르소나를 채용하셔야 모두가 행복할 수 있습니다. 초기에 데이터 마트 만드는 일, 대시보드 만드는 일이 많이 선행되어야 하기 때문에 이 쪽 능력이 우수하신 분을 뽑는게 적합합니다.

## 결론 
이상 결론 지어 답변을 드리면, 

1. 데이터 엔지니어링 가능 인력이 있으며 (꼭 데이터 엔지니어가 아니더라도 역할을 할 수 있어야 함)
2. 분석가가 분석을 할 수 있는 최소한의 환경이 준비되어 있을 때 (위에 조건 명시)

조직에 첫 데이터 분석가가 필요할 것 같습니다.
